api/grant.go

```diff
@@ -10,21 +10,21 @@ import (
 )
 
 type Grant struct {
-	ID uid.ID `json:"id"`
+	ID uid.ID `json:"id" note:"ID of grant created" example:"3w9XyTrkzk"`
 
 	Created   Time   `json:"created"`
 	CreatedBy uid.ID `json:"created_by" note:"id of the user that created the grant"`
 	Updated   Time   `json:"updated"`
 
-	User      uid.ID `json:"user,omitempty"`
-	Group     uid.ID `json:"group,omitempty"`
-	Privilege string `json:"privilege" note:"a role or permission"`
-	Resource  string `json:"resource" note:"a resource name in Infra's Universal Resource Notation"`
+	User      uid.ID `json:"user,omitempty" note:"UserID for a user being granted access" example:"6hNnjfjVcc"`
+	Group     uid.ID `json:"group,omitempty" note:"GroupID for a group being granted access" example:"3zMaadcd2U"`
+	Privilege string `json:"privilege" note:"a role or permission" example:"cluster-admin"`
```

Comment:
```suggestion
	Privilege string `json:"privilege" note:"a role or permission" example:"admin"`
```

A suggestion where cluster-admin is an uncommon case.